src/TypeScript/DefinitionWriter.cpp

```diff
@@ -654,7 +698,7 @@ std::string DefinitionWriter::tsifyType(const Type& type)
             return "string";
         }
         else if (interface.name == "NSDate") {
```

Comment:
The iOS runtimes implicitly marshalls `NSDate` instances to JS `Date` ones, so I think this should stay.